modules/ROOT/pages/_partials/app-security/proc-calling-the-app-security-clientInit.adoc

```diff
@@ -22,7 +22,13 @@ appSecurity.clientInit()
     from the server set by the admin.
   */
 }).catch(err  => {
-    // Login error
+    /**
+      You can handle errors connecting the the 
```

Comment:
they were a great band back in the day (https://en.wikipedia.org/wiki/The_The)

:)

Reply:
:)